src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRequest.cs

```diff
@@ -132,16 +132,15 @@ public class RestoreRequest
         public ProjectStyle ProjectStyle { get; set; } = ProjectStyle.Unknown;
 
         /// <summary>
-        /// Restore output path
+        /// RestoreOutputPath, which is where the restore output (assets file and generated props and targets) will go
         /// </summary>
         public string RestoreOutputPath { get; set; }
 
         /// <summary>
-        /// Base Intermediate output path
+        /// AssetsCachePath, which is where the no-op restore cache will go
         /// </summary>
-        public string BaseIntermediateOutputPath { get; set; }
+        public string AssetsCachePath { get; set; }
```

Comment:
This is also a NuGet restore artifact just like assets file or target file, so I think this should also go in the same location.